README.md

```diff
@@ -301,6 +301,18 @@ private void ConfigureAuthenticationTokenHandler(IServiceCollection services)
 }
 ```
 
+You will also need to provide a `DelegatingHandler` to your endpoints if you want to handle the access token logic through the `AuthenticationTokenHandler` provided.
```

Comment:
This isn't specific to `AuthenticationtokenHandler`. All the DelegatingHandlers shown in this page need to be added in some way to the desired `HttpClient`.